Description: Ensure make uses bash as its shell
 The upstream Makefile contains several bash constructs (like '[['). These
 fail on debian because the default shell for make is /bin/sh which points to
 dash. And dash does not support such constructs. So have make use /bin/bash
 as its shell.
Author: Alan M Varghese
Last-Update: 2024-06-15
Forwarded: not-needed
--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,5 @@
 PREFIX = /usr/local
+SHELL = /bin/bash
 
 legacyrenderer:
 	cmake --no-warn-unused-cli -DCMAKE_BUILD_TYPE:STRING=Release -DCMAKE_INSTALL_PREFIX:STRING=${PREFIX} -DLEGACY_RENDERER:BOOL=true -S . -B ./build -G Ninja
